### ISSUE TITLE
bug: incorrect indefinition of arguments in function

### ISSUE THREAD

**Issue by kitsonk** [denoland/deno_lint#320]
In functions, `arguments` is being flagged as being undefined... 😢 

For example:

```ts
export function foo(a: string, b?: string): void {
  if (arguments.length === 1) {
    console.log("one", a);
  } else {
    console.log("two", a, b);
  }
}
```

Results in:

```
(no-undef) arguments is not defined
  if (arguments.length === 1) {
      ^^^^^^^^^
    at /example.ts:2:6
```

**Comment by kitsonk** [comment]
Duplicate of #317
